### ISSUE TITLE
Add support for the openfoodfacts-events API (leaderboard, badge and gamification system)

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-dart#349]
### What
- [ ] Add support for the badge and event system
  - Source: https://github.com/openfoodfacts/openfoodfacts-events
  - API documentation: https://events.openfoodfacts.org/docs / https://events.openfoodfacts.net/docs
### Why
 - Will unblock openfoodfacts/smooth-app/issues/20

**Comment by jasmeet0817** [comment]
Yeah, i plan to retake this one up but someone in mid January, anyways i think this lower priority than some of the other stuff

**Comment by teolemon** [comment]
Did you start working on any of the API routes, or is that something a community member could start working on for the DART package potentially ?

**Comment by teolemon** [comment]
I clarified that there's no code dependancy between the 2 in the initial issue. If a community member would like to pick it up, so that we can demo getting one badge (image taking) and showing it in Smoothie, at the Fellowship wrapup event, that would be great.
